sunpy/instr/tests/test_aia.py

```diff
@@ -16,6 +16,8 @@
 def test_aiaprep():
     # Test that header info for the map has been correctly updated
     # Check all of these for Map attributes and .meta values?
+    # Check array shape
+    assert prep_map.data.shape == original.data.shape
```

Comment:
Any suggestions on how to do that?  Just checking the equality of the dimensions wouldn't be sufficient.  Rather than raising an error, how about a warning and document that aiaprep should be used before anything else?